README.md

```diff
@@ -34,10 +34,18 @@ If you plan to use `Makie#master`, you likely also need to check out `AbstractPl
 ## Dependencies
 You will need to have ffmpeg in the path to run the video recording examples.
 On linux you also need to add the following to get GLFW to build (if you don't have those already):
+
+### Debian/Ubuntu
 ```
 sudo apt-get install ffmpeg cmake xorg-dev
 ```
 
+### RedHat/Fedora
+```
+sudo dnf install ffmpeg cmake libXrandr-devel libXinerama-devel libXcursor-devel
+```
+Note that the [RPM Fusion repo](https://rpmfusion.org/) is needed for `ffmpeg`.
```

Comment:
Might be useful to include how to add that repo, so that users can copy, paste, and install directly.

Reply:
This is what Fedora's docs say:

```bash
sudo dnf install \
  https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm
```

Reply:
I'd leave users to follow the link since there are legal reasons why it's in a separate repo. Fedora (& RedHat) are quite hot on this and so it'd say it's generally better to for users to make an informed decision by reading the RPM Fusion site (and they do make it pretty trivial to do once you've looked at the site).

Reply:
Makes sense, I wasn't aware that this was a hot topic.